src/NumericalAlgorithms/LinearOperators/Mass.hpp

```diff
@@ -0,0 +1,30 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function mass.
```

Comment:
This comment does not clearly tell me what this code is supposed to do. Can you please clarify?